tests/scale/scale_trafficSplit_test.go

```diff
@@ -0,0 +1,278 @@
+		// Framework data handle and hook to compute results
+		// We wait to initialize it till prom/grafana instances are available in OSM's case.
+		var sd *DataHandle
+
+		AfterEach(func() {
+			if sd != nil {
+				sd.WrapUp()
+			}
+		})
+
+		It("Tests HTTP traffic from Clients to the traffic split Cluster IP", func() {
+			// Install OSM with all the requirements
+			var err error
+			sd, err = scaleOSMInstall()
+			Expect(err).To(BeNil())
+
+			// -- test specific data below
+
+			const (
+				// to name the header we will use to identify the server that replies
+				HTTPHeaderName = "podname"
+
+				// Base names for the resources we will generate
+				clientNamePrefix      = "client"
+				serverNamespacePrefix = "server"
+				TrafficSplitPrefix    = "traffic-split"
+			)
+
+			var (
+				// Number of client services per iteration
+				numberOfClientServices = 2
+				clientReplicaSet       = 5
+
+				// Number of server services per iteration
+				numberOfServerServices = 5
+				serverReplicaSet       = 2
+
+				// Total number of created resources, also used to name the resources we generate
+				totalNumberOfClientServices   = 0
+				totalNumberOfServerServices   = 0
+				totalNumberOfServerNamespaces = 0
+
+				// Used across the test to wait for concurrent steps to finish
+				wg = sync.WaitGroup{}
+			)
+
+			// Scale loop
+			sd.Iterate(func() {
+				// The following section computes the clients and servers (quantity and names) to be run this iteration
+				clientServices := []string{}
+				serverServices := []string{}
+				allNamespaces := []string{}
+
+				// All servers services for a traffic split live in the same namespace
+				serverNamespace := fmt.Sprintf("%s-%d", serverNamespacePrefix, totalNumberOfServerNamespaces)
+				totalNumberOfServerNamespaces++
+
+				for i := totalNumberOfClientServices; i < (totalNumberOfClientServices + numberOfClientServices); i++ {
+					clientServices = append(clientServices, fmt.Sprintf("%s-%d", clientNamePrefix, i))
+				}
+				totalNumberOfClientServices += numberOfClientServices
+
+				for i := totalNumberOfServerServices; i < (totalNumberOfServerServices + numberOfServerServices); i++ {
+					serverServices = append(serverServices, fmt.Sprintf("%s-%d", serverNamespace, i))
+				}
+				totalNumberOfServerServices += numberOfServerServices
+
+				allNamespaces = append(allNamespaces, clientServices...)
+				allNamespaces = append(allNamespaces, serverNamespace)
+
+				// Create the NSs for the services we just computed
+				Expect(Td.CreateMultipleNs(allNamespaces...)).To(Succeed())
+				Expect(Td.AddNsToMesh(true, allNamespaces...)).To(Succeed())
+
+				// Create server services
+				for _, serverApp := range serverServices {
+					svcAccDef, deploymentDef, svcDef := Td.SimpleDeploymentApp(
+						SimpleDeploymentAppDef{
+							Name:         serverApp,
+							Namespace:    serverNamespace,
+							ReplicaCount: int32(serverReplicaSet),
+							Image:        "simonkowallik/httpbin",
```

Comment:
The image we use in most other tests is `kennethreitz/httpbin`. Probably worth settling on one.

Reply:
I think original httpbin didn't have the `env to header` feature we are using to verify the pod which answered a request...
https://github.com/simonkowallik/httpbin